pkg/nodeconfig/payload/payload.go

```diff
@@ -20,6 +20,11 @@ const (
 	// KubeProxyPath contains the path of the kube-proxy binary. The container image should already have this binary
```

Comment:
If this commit was co-authored with @selansen , please add `co-authored-by` to the commit message.